vm/systemSmartContracts/governance.go

```diff
@@ -1005,13 +1038,40 @@ func (g *governanceContract) proposalContainsVoter(proposal *GeneralProposal, vo
 // computeVotingPower returns the voting power for a value. The value can be either a balance or
 //  the staked value for a validator
 func (g *governanceContract) computeVotingPower(value *big.Int) (*big.Int, error) {
-	if value.Cmp(big.NewInt(0)) == -1 {
+	if value.Cmp(zero) == -1 {
 		return nil, fmt.Errorf("cannot compute voting power on a negative value")
 	}
 
 	return big.NewInt(0).Sqrt(value), nil
 }
 
+// computeAccountLeveledPower takes a value and some voter data and returns the voting power of that value in
+//  the following way: the power of all votes combined has to be sqrt(sum(allAccountVotes)). So, the new
+//  vote will have a smaller power depending on how much existed previously
+func (g *governanceContract) computeAccountLeveledPower(value *big.Int, voteData *VoteSet) (*big.Int, error) {
+	totalBalance := big.NewInt(0)
+	for _, accountVote := range voteData.VoteItems {
+		if accountVote.Type != Account {
+			continue
+		}
+
+		totalBalance.Add(totalBalance, accountVote.Balance)
```

Comment:
as an optimization, this can be kept in the voteData struct and computed each time we add a new vote item.

Reply:
done